utils/cache.ts

```diff
@@ -25,11 +25,11 @@ export class Cache<TData, TInput> {
       this.data = await this.getData(input)
       this.cachedTime = currTime
 
-      console.log('Returning data from API:', this.apiRoute)
+      console.log('Returning data from API:', this.apiRoute) // eslint-disable-line no-console
       return this.data
     }
 
-    console.log('Returning cached data:', this.apiRoute)
+    console.log('Returning cached data:', this.apiRoute) // eslint-disable-line no-console
```

Comment:
Looking from server side world, any way to log this in a way that isn't directly logging to console? Or only log with a debug flag? I am kind of on the side of the linter

Reply:
Tbh I'm not even sure where these logs end up in our staging + prod environments (some AWS Amplify console?) but I agree we should look into setting up real logging. [Next has some docs on it](https://nextjs.org/docs/going-to-production#logging) we can maybe follow.